<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addSports</class>
 <widget class="QWidget" name="addSports">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>424</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>添加球票</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/Qt-picture/MDLQ.jpeg</normaloff>:/Qt-picture/MDLQ.jpeg</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(172, 224, 249);</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文楷体</family>
     <pointsize>16</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;华文楷体&quot;;</string>
   </property>
   <property name="text">
    <string>比赛类型</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文楷体</family>
     <pointsize>16</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;华文楷体&quot;;</string>
   </property>
   <property name="text">
    <string>比赛名称</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文楷体</family>
     <pointsize>16</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;华文楷体&quot;;</string>
   </property>
   <property name="text">
    <string>比赛时间</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>280</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文楷体</family>
     <pointsize>16</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;华文楷体&quot;;</string>
   </property>
   <property name="text">
    <string>比赛场地</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>340</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文楷体</family>
     <pointsize>16</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;华文楷体&quot;;</string>
   </property>
   <property name="text">
    <string>比赛主场</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>420</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文楷体</family>
     <pointsize>16</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;华文楷体&quot;;</string>
   </property>
   <property name="text">
    <string>价格</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>420</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文楷体</family>
     <pointsize>16</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;华文楷体&quot;;</string>
   </property>
   <property name="text">
    <string>票数</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="sportsType">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>90</y>
     <width>160</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;Stencil&quot;;</string>
   </property>
   <item>
    <property name="text">
     <string>sport1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>sport2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>sport3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>sport4</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>sport5</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>sport6</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="gamename">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>150</y>
     <width>240</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{ border-radius:6; border:1px solid white}</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="gameTime">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>220</y>
     <width>240</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{ border-radius:6; border:1px solid white}</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="gamePlace">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>280</y>
     <width>240</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{ border-radius:6; border:1px solid white}</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="gameHost">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>340</y>
     <width>240</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{ border-radius:6; border:1px solid white}</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="ticketPrice">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>420</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="maximum">
    <number>10000</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="ticketCount">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>420</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="maximum">
    <number>10000</number>
   </property>
  </widget>
  <widget class="QPushButton" name="okBtn">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>490</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{ border:1px solid #fff1eb; border-radius:6; 
background-color:#fff1eb; font: 16pt &quot;华文楷体&quot;;}
QPushButton:hover{border:2px solid black;}</string>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="noBtn">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>490</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{ border:1px solid #fff1eb; border-radius:6; 
background-color:#fff1eb; font: 16pt &quot;华文楷体&quot;;}
QPushButton:hover{border:2px solid black;}</string>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QPushButton" name="clearBtn">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>490</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{ border:1px solid #fff1eb; border-radius:6; 
background-color:#fff1eb; font: 16pt &quot;华文楷体&quot;;}
QPushButton:hover{border:2px solid black;}</string>
   </property>
   <property name="text">
    <string>清空</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文楷体</family>
     <pointsize>16</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">
font: 16pt &quot;华文楷体&quot;;</string>
   </property>
   <property name="text">
    <string>比赛编号</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="gamenumber">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>40</y>
     <width>240</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{ border-radius:6; border:1px solid white}</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
